src/Uno.Extensions.Authentication.MSAL/build/Package.targets

```
@@ -36,4 +36,16 @@
 		</ItemGroup>
 		<Message Importance="high" Text ="Removed: @(_WebView2CoreOutputsToExclude)" />
 	</Target>
+
+	<Target Name="_ValidateMsalDependencyWasm"
+			BeforeTargets="Compile"
+			Condition="'$(IsUnoHead)'=='true' and '$(UnoRuntimeIdentifier)'=='WebAssembly'">
```

```suggestion
			Condition="'$(IsMSALSupported)'=='true'>
```
Probably best to validate this against any MSAL Supported target.

---

Would we need to change anything here?

https://github.com/unoplatform/uno.extensions/blob/6650d489541bda3a219e3ec84cc4b0999bd230e4/src/Uno.Extensions.Authentication.MSAL/build/Package.targets#L6-L9

---

No need to change that... it's already got the conditions to determine if the platform is supported or not. That's why using the `IsMSALSupported` is the better variable to check.

---

I've changed the condition to be `'$(IsUnoHead)'=='true' and '$(UnoRuntimeIdentifier)'=='WebAssembly'`.

---

@jeromelaban are you able to confirm this change is as expected? I'm ok with this, so happy for it to be approved and merged.